roamance/src/components/social/SideNavigation.tsx

```
@@ -0,0 +1,320 @@
+interface SideNavigationProps {
+  activeTab: string;
+  setActiveTab: (tab: string) => void;
+}
+
+export function DesktopSideNavigation({
+  activeTab,
+  setActiveTab,
+}: SideNavigationProps) {
+  return (
+    <aside className="hidden md:block w-72 flex-shrink-0 relative">
+      <div className="absolute top-8 left-4 w-16 h-16 bg-gradient-to-br from-purple-200/20 to-indigo-300/20 dark:from-purple-800/10 dark:to-indigo-700/10 rounded-full blur-xl z-0"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)